web/src/store/travel-time/actions.ts

```diff
@@ -40,7 +40,7 @@ export function getTravelTimes(travels: TravelTimeAbstraction[]) {
 		},
 		range: {
 			enabled: true,
-			width: 2
+			width: 7200
```

Comment:
Why do we need a fixed range here?

Reply:
The default range is 30 minutes when it's not supplied in the body, in order to give a better travel range with public transit this parameter insures that all travel times within a 2 hour span are calculated.

So if a train departs at 9:31 at location A and arrives at location B at 9:44 this normally would not be included in a 15 minute range search. Now it will.

Reply:
Perfect, thanks for the explanation!